README.md

```diff
@@ -2,140 +2,167 @@
+```python
+import runhouse as rh
 
-* 🦸‍♀️ **OSS maintainers** who want to improve the accessibility, reproducibility, and reach of their code,
-without having to build support or examples for every cloud or compute system (e.g. Kubernetes) one by one.
-   * See this in action in 🤗 Hugging Face ([Transformers](https://github.com/huggingface/transformers/blob/main/examples/README.md#running-the-examples-on-remote-hardware-with-auto-setup), [Accelerate](https://github.com/huggingface/accelerate/blob/main/examples/README.md#simple-multi-gpu-hardware-launcher)) and 🦜🔗 [Langchain](https://python.langchain.com/en/latest/modules/models/llms/integrations/runhouse.html)
-* 👩‍🔬 **ML Researchers and Data Scientists** who don't want to spend or wait 3-6 months translating and packaging
-their work for production.
-* 👩‍🏭 **ML Engineers** who want to be able to update and improve production services, pipelines, and artifacts with a
-Pythonic, debuggable devX.
-* 👩‍🔧 **ML Platform teams** who want a versioned, shared, maintainable stack of services and data artifacts that
-research and production pipelines both depend on.
+def run_home(name: str):
+    return f"Run home {name}!"
 
-### 🦾 How does it work?
+if __name__ == "__main__":
+    cpu_box = rh.ondemand_cluster(name="my-cpu", instance_type="CPU:2", provider="aws")
+    remote_fn = rh.function(run_home).to(cpu_box)
+    print(remote_fn("Jack"))
+```
 
-_"Learn once, run anywhere"_
+## 🤔 Why?
+
+Runhouse is built to do four things:
+1. Make it easy to send an arbitrary block of your code - function, subroutine, class, generator, whatever -
+to run on souped up remote infra. It's basically a flag flip.
+1. Eliminate CLI and Flask/FastAPI boilerplate by allowing you to send your function or class directly to your remote
+infra to execute or serve, and keep them debuggable like the original code, not a subprocess.Popen or postman/curl call.
+1. Bake-in the middleware and automation to make your app production-quality, secure, and sharable instantly.
+That means giving you best-of-breed auth, HTTPS, telemetry, packaging, and deployment automation, with ample
+flexibility to swap in your own.
+1. Bring the power of Ray to any app, anywhere, without having to learn Ray or manage Ray clusters, like Next.js did
+for React. OpenAI, Uber, Shopify, and many others use Ray to power their ML infra, and Runhouse makes its best-in-class
+features accessible to any project, team, or company.
+
+## 🤨 Who is this for?
+
+* 👩‍🔧 **Engineers, Researchers and Data Scientists** who don't want to spend 3-6 months translating and packaging
+their work to share it, and want to be able to iterate and improve production services, pipelines, and artifacts
+with a Pythonic, debuggable devX.
+* 👩‍🔬 **ML and data teams** who want a versioned, shared, maintainable stack of services used across
+research and production.
+* 🦸‍♀️ **OSS maintainers** who want to supercharge their setup flow by providing a single script to stand up their app
```

Comment:
Not sure if prototypers is a segment we want to mention, but Runhouse is definitely great at that.

"Serve your own instance of the newest open source model on any compute with X lines of code."